tests/README.md

```diff
@@ -28,21 +28,27 @@ poetry install
 When running `pytest`, these environment variables must be set in your environment. You may prefix your command with the environment variables as well. For example: `SINDRI_API_KEY=your_api_key pytest`
 
 - `SINDRI_API_KEY=your_api_key` *Required*
-- `SINDRI_API_URL=https://sindri.app/api` *Optional. Default value shown*
+- `SINDRI_BASE_URL=https://sindri.app` *Optional. Default value shown*
 
 ### Run Pytest
 You must be in the root directory of this repo: `cd ..`
 
-Here are several options for running tests with environment variables:
+Here are several options for running the pytest unit tests with environment variables:
 
-Exporting ENVs:
+#### Exporting ENVs
 ```bash
 export SINDRI_API_KEY=your_api_key
-export SINDRI_API_URL=https://sindri.app/api
+export SINDRI_BASE_URL=https://sindri.app
 pytest
 ```
 
-Prefixing ENVs:
+#### Prefixing ENVs
 ```bash
-SINDRI_API_KEY=your_api_key SINDRI_API_URL=https://sindri.app/api pytest
+SINDRI_API_KEY=your_api_key SINDRI_BASE_URL=https://sindri.app pytest
+```
+
+#### Prefixing ENVs for Sindri internal developers
+If you are running the Sindri api locally at `~/forge` with `~/forge/API_KEY` file populated with a valid api key, you can invoke the `pytest` unit tests to hit your local Sindri API instance with:
+```bash
+SINDRI_BASE_URL=http://localhost SINDRI_API_KEY=$(cat ~/forge/API_KEY) pytest
```

Comment:
The use of the "forge" name is deprecated AFAIK. Perhaps use a different name here for the example?

Reply:
I'm open to suggestions. I marked this section of the readme as `for Sindri internal developers` so maybe that's enough? feel free to change it

Reply:
Changed `forge` to `myproject` in [d346c19](https://github.com/Sindri-Labs/sindri-python/pull/39/commits/d346c1973c106e46f67c58055d2f7fc26ccf4226).